recipes/hmftools-redux/redux.sh

```
@@ -0,0 +1,69 @@
+	    else
+                pass_args="$pass_args \"$arg\"" #quotes later arguments to avoid problem with ()s in MarkDuplicates regex arg
+            fi
+            ;;
+    esac
+done
+
+if [ "$jvm_mem_opts" == "" ]; then
+    jvm_mem_opts="$default_jvm_mem_opts"
+fi
+
+pass_arr=($pass_args)
```

Shellcheck

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)